### PR TITLE
Added Violet Magic theme

### PR DESCRIPTION
**Pull request by Fluffy-hyena**
no shrek theme :(

### CODE REVIEW

File: plugins/Violet Magic theme/manifest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	"author": "Fluffy hyena",
+	"name": "Violet Magic Theme",
+	"version": "1.0.0",
+	"id": "d4a11ede-ec96-49af-ab3c-f0f3857d8d8d",
+	"description": "Adds a nice looking theme to bridge (looks better in dark theme)",
+	"api_version": 1,
+	"tags": [
+		"Theme"
+	],
+	"releaseTimestamp": 1613568421816
```
Comment (solvedDev) [inline plugins/Violet Magic theme/manifest.json:11]: Please remove the `releaseTimestamp` from your manifest

File: plugins/Violet Magic theme/manifest.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+	"author": "Fluffy hyena",
+	"name": "Violet Magic Theme",
+	"version": "1.0.0",
+	"id": "d4a11ede-ec96-49af-ab3c-f0f3857d8d8d",
```
Comment (solvedDev) [inline plugins/Violet Magic theme/manifest.json:5]: Did you generate a new UUID?

**Review by solvedDev — CHANGES_REQUESTED** [review]

File: plugins/Violet Magic theme/manifest.json (hunk 2 above)
Comment (Fluffy-hyena) [reply plugins/Violet Magic theme/manifest.json:5]: Yeah

File: plugins/Violet Magic theme/manifest.json (hunk 1 above)
Comment (Fluffy-hyena) [reply plugins/Violet Magic theme/manifest.json]: Done

**Review by solvedDev — APPROVED** [review]
👍

**Pull request merged**